### PR TITLE
LPS-171822 User facet 

### PR DESCRIPTION
**Pull request by joshuacords**
https://liferay.atlassian.net/browse/LPS-171822

Code based on https://github.com/arboliveira/liferay-portal/pull/522/commits

**Authors:** @joshuacords, @BryanEngler

### CODE REVIEW

**Comment by joshuacords** [comment]
ci:test:relevant

**Comment by joshuacords** [comment]
ci:test:sf

**Comment by joshuacords** [comment]
ci:test:search

**Comment by joshuacords** [comment]
ci:test:sf

**Comment by joshuacords** [comment]
ci:test:search

**Comment by joshuacords** [comment]
Note: functional tests need to be updated before forwarding.

**Review by gustavolimav — APPROVED** [review]
The changes looks good to me and I tested them locally. 🚀

**Comment by joshchong** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/facet/user/UserFacetSearchContributor.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public interface UserFacetBuilder {
 
 		public UserFacetBuilder maxTerms(int maxTerms);
 
-		public UserFacetBuilder selectedUserNames(String... selectedUserNames);
+		public UserFacetBuilder selectedUserIds(long... selectedUserNames);
```
Comment (BryanEngler) [inline modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/facet/user/UserFacetSearchContributor.java:33]: update param name too?

**Comment by joshuacords** [comment]
ci:test:sf

File: modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/facet/user/UserFacetSearchContributor.java (hunk 1 above)
Comment (joshuacords) [reply modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/facet/user/UserFacetSearchContributor.java]: Thanks Bryan

**Comment by joshuacords** [comment]
Given the simple renaming, I don't see a need to rerun tests outside of sf.

@joshchong - be sure to add the new commit to the final PR.

**Comment by joshchong** [comment]
Sent to BChan:
https://github.com/brianchandotcom/liferay-portal/pull/140069

**Pull request closed without merging**